packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2744,6 +2751,16 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         }
     }
 
+    private verifyCanSubmitOps() {
+        if (this.mc.config.getBoolean("Fluid.ContainerRuntime.DisableOpReentryCheck") === true) {
+            return;
+        }
+
+        if (this.runWithoutOpsCalls > 0) {
+            throw new UsageError("Op was submitted from within a `runWithoutOps` callback");
```

Comment:
That error alone is not detailed enough to help most customers understand what happened and what they need to do.
I think we need to find a way (either via comments in the code, readme.md in the package, something else, or combo of all the above) to let OCEs and developers (internal and external) know what to do next.